client/ratings_reviews/helpers.js

```diff
@@ -0,0 +1,75 @@
+const formatDate = (dateString) => {
+  var [year, month, day] = dateString.split('-');
+
+  if (month === '01') {
```

Comment:
you could turn this into an object...{"1": "January"} etc...

Reply:
Fantastic idea! Will make changes accordingly